src/fonts/ionicons.scss

```diff
@@ -8,6 +8,7 @@ $ionicons-font-path: $font-path !default;
 
 @import "ionicons-icons";
 @import "ionicons-variables";
+@import "~hail2u-scss-functions/string/_str-replace";
```

Comment:
I'm not sure about this for icons. I wonder if there is a better way to decide if the icon should or shouldn't be flipped on rtl. We're thinking maybe we could add some metadata to the icon itself and look at it that way? We need to discuss this more but maybe this could be split out from this PR for further discussion.